app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary.mdx

```diff
@@ -0,0 +1,110 @@
+---
+meta:
+  title: Glossary
+  description: List of terms that we use in the Virtual Coffee community.
+hero:
+  Hero: UndrawReadingList
+tags: [memberresources]
+order: 11
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+In this article, you can find the meaning of terms we use in the Virtual Coffee community.
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## Virtual Coffee Related
+
+### Coffee(s)
+
+Our weekly Zoom chats | Tuesdays at 9:00 AM ET and Thursdays at 12:00 PM ET.
+
+### Coffee Table Groups
+
+They are small, special interest groups created and run by our members. They can involve Zoom meetings, async Slack hangouts, or anything else the members would like to do.
+
+### Maintainers
+
+Virtual Coffee's <Link to="/members">core team</Link>.
+
+### Note Taker
+
+An assigned member who takes notes during a Coffee and shares them afterward on Slack.
+
+### Room Leader
+
+An assigned member who facilitates a Coffee <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#breakout-rooms">breakout room</Link> by moderating the conversations during the Coffee.
+
+### VC
+
+The abbreviation of Virtual Coffee. Our members usually refer to us as VC.
+
+</TextContainer>
+
+<TextContainer>
+
+## Virtual Coffee Events Related
+
+### Hacktoberfest Initiative
+
+An event where we encourage, help, and support our members to contribute to and/or maintain open sources during <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#hacktoberfest">Hacktoberfest</Link>.
+
+### Lightning Talk
+
+Short (5-20 minute time range) tech-related presentations from members to members. We hold this event occasionally.
+
+### Lunch & Learns
+
+It's a (typically) hour-long speaking opportunity for anyone in the community to share their thoughts and ideas with a small audience of friends.
+
+### Monthly Challenge
+
+Every month, we create a challenge with a particular theme to allow our members to learn, grow, and receive support and mentorship.
+
+</TextContainer>
+
+<TextContainer>
+
+## General
+
+### Breakout Rooms
+
+Small groups in Zoom meetings.
+
+### COC
+
+The abbreviation of Code of Conduct.
+
+### Hacktoberfest
+
+[A month-long virtual event](https://hacktoberfest.com/) that encourages and supports open source contributions, sponsored by DigitalOcean.
+
+### Office Hours
+
+A pre-arranged time when a person is available during working hours to answer questions or provide assistance. <br />
+_Credit: [Wordnik](https://www.wordnik.com/words/office%20hours)_
+
+### Slack
+
+The chat service that we use for our members to connect.
+
+### Zoom
+
+The communication and collaboration platform that we use for our virtual meetings and events.
```

Comment:
[Zoom](https://zoom.us/) is the communication and collaboration platform that we use for our virtual meetings and events.